### PR TITLE
Add Feedback link in menu

### PR DESCRIPTION
**Pull request by kenancasey**

### CODE REVIEW

File: src/components/menu/Menu.tsx [hunk 1]
```diff
@@ -18,13 +18,13 @@ export const Menu = ({ showMenu, setShowMenu }: props) => {
 
   const ref = useDetectClickOutside({ onTriggered: closeMenu })
   const menuItemClasses =
-    'flex origin-left transform text-xl md:text-2xl font-semibold text-black transition-all delay-100 duration-300 hover:scale-125 hover:text-white dark:text-white hover:dark:text-sky-400'
+    'flex origin-left transform text-sm md:text-2xl font-semibold text-black transition-all delay-100 duration-300 hover:scale-105 hover:text-white dark:text-white hover:dark:text-sky-400'
```
Comment (kadenbking) [inline src/components/menu/Menu.tsx:21]: On mobile, I'd prefer for the menu items to be a little larger. If `xl` is too big, then perhaps `md` or `lg` as oppose to `sm`

Comment (kenancasey) [reply src/components/menu/Menu.tsx:21]: Good call. Let me try that.

Comment (kenancasey) [reply src/components/menu/Menu.tsx]: I've updated that. What do you think?

**Review by kadenbking — APPROVED** [review]
LGTM

**Pull request merged**